marlowe-runtime/doc/resources.md

```diff
@@ -0,0 +1,55 @@
+# Recommended Resources for Deploying Marlowe Runtime
+
+This document summarizes recommendations for the computational resources required to deploy Marlowe Runtime on public networks (`mainnet`, `preprod`, and `preview`).
+
+
+## Marlowe Services
+
+The table below provides guidance on the memory and CPU resources required for Marlowe Runtime's services. These are based on peak usages over the following disparate circumstances across all three public networks:
+
+1. Indexing the blockchain from genesis to the tip.
+2. Running 500 Marlowe contracts in parallel at the maximum possible transaction-building and -submission rate.
+3. Querying the history of all Marlowe contracts at the maximum rate.
+
+| Service                 | Memory  | CPU (cores) |
+|-------------------------|--------:|------------:|
+| `marlowe-chain-indexer` | 2500 MB |        50 % |
+| `marlowe-chain-sync`    |  100 MB |        50 % |
+| `marlowe-indexer`       |  100 MB |        10 % |
+| `marlowe-sync`          |  500 MB |        50 % |
```

Comment:
Actually, I just ran eight copies of `marlowe-finder` and that pushes `marlowe-sync`'s memory usage above 4 GB. I'm investigating further in order to understand its scaling behavior.

Reply:
Here is the situation with `marlowe-sync`.
![marlowe-sync](https://user-images.githubusercontent.com/2235898/226071358-ec825da6-fb14-4c5e-8692-0da52ed035ab.png)

Reply:
We verified that this is an artifact of too-verbose logging and is not an intrinsic limitation of `marlowe-sync`.